base/setup/reactos/lang/fi-FI.rc

```diff
@@ -17,210 +17,211 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYS
 CAPTION "ReactOS Setup"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    AUTORADIOBUTTON "Install ReactOS", IDC_INSTALL, 7, 7, 277, 10, WS_GROUP | WS_TABSTOP
-    LTEXT "Install a new copy of ReactOS. This option does not keep your files, settings and programs. You can make changes to disks and partitions.", IDC_INSTALLTEXT, 18, 18, 269, 25, NOT WS_GROUP
-    AUTORADIOBUTTON "Update or repair an installed ReactOS", IDC_UPDATE, 7, 50, 277, 10
-    LTEXT "Update or repair an installed copy of ReactOS. This option keeps your files, settings and programs. This option is only available if ReactOS is already installed on this computer.", IDC_UPDATETEXT, 18, 61, 269, 30, NOT WS_GROUP
-    LTEXT "Click Next to setup the devices.", IDC_STATIC, 7, 128, 297, 8
+    AUTORADIOBUTTON "Asenna ReactOS", IDC_INSTALL, 7, 7, 277, 10, WS_GROUP | WS_TABSTOP
+    LTEXT "Asenna uusi ReactOS kopio. Tämä vaihtoehto ei säilytä tiedostojasi, asetuksiasi ja sovelluksia. Voit tehdä muutoksia levyihin ja osiohin.", IDC_INSTALLTEXT, 18, 18, 269, 25, NOT WS_GROUP
+    AUTORADIOBUTTON "Päivitä tai korjaa ReactOS-asennus", IDC_UPDATE, 7, 50, 277, 10
+    LTEXT "Päivitä tai korjaa ReactOS-asennus. Tämä vaihtoehto säilyttää tiedostot, asetukset ja sovellukset. Tämä vaihtoehto on vain 
+käytettävissä, jos ReactOS on asennettu tietokoneelle.", IDC_UPDATETEXT, 18, 61, 269, 30, NOT WS_GROUP
```

Comment:
mandatory change:
```suggestion
käytettävissä, jos ReactOS on asennettu tietokoneelle.", IDC_UPDATE, 18, 61, 269, 30, NOT WS_GROUP
```
An oopsie currently prevents the compilation:
![image](https://github.com/user-attachments/assets/a3f0ac69-02f8-4626-919f-989cb042a530)

Reply:
This is resolved. But to succeed the compilation entirely also the 2nd thing brought up by Hermes needs to be addressed.